android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -302,24 +312,15 @@ public void captureEnvelope(String envelope, Promise promise) {
             } else {
                 File installation = new File(outboxPath, UUID.randomUUID().toString());
                 try (FileOutputStream out = new FileOutputStream(installation)) {
-                    out.write(envelope.getBytes(Charset.forName("UTF-8")));
+                    out.write(bytes);
                 }
             }
         } catch (Throwable ignored) {
-            logger.severe("Error reading envelope");
+            logger.severe("Error while writing envelope to outbox.");
         }
         promise.resolve(true);
     }
 
-    @ReactMethod
-    public void getStringBytesLength(String payload, Promise promise) {
-        try {
-            promise.resolve(payload.getBytes("UTF-8").length);
-        } catch (UnsupportedEncodingException e) {
-            promise.reject(e);
-        }
-    }
-
```

Comment:
I've removed this method since the raw bytes of the envelope are assembled in js, the bytes length have to be correct in the js.

Reply:
You need to guarantee that this works tho, this was a bug multiple times in multiple places.

Reply:
Is anywhere issue, where are some steps how to reproduce the issue, so I can double check if it works now.

Reply:
You can use git blame and track when and why the code was added, https://github.com/getsentry/sentry-react-native/pull/773 has more context.

Reply:
Thanks, will do next time. 

I've added a test for the byte size of the payload. There should be no trouble since we assemble the raw bytes in JS now and then pass them to the native layers.